scripts/pi-hole/js/db_queries.js

```diff
@@ -77,7 +77,7 @@ $(function () {
           moment().subtract(1, "month").endOf("month"),
         ],
         "This Year": [moment().startOf("year"), moment()],
-        "All Time": [moment(0), moment()],
+        "All Time": [moment(mintimestamp), moment()],
```

Comment:
`mintimestamp` doesn't exist in this file. `mintimestamp` is declared in `db_graph.js` file.

You are editing the wrong file.